chia/server/ws_connection.py

```diff
@@ -696,13 +702,15 @@ async def _read_one_message(self) -> Optional[Message]:
                 message_type = ProtocolMessageTypes(full_message_loaded.type).name
             except Exception:
                 message_type = "Unknown"
-            if not self.inbound_rate_limiter.process_msg_and_check(
+            accepted, limiter_msg = self.inbound_rate_limiter.process_msg_and_check(
                 full_message_loaded, self.local_capabilities, self.peer_capabilities
-            ):
+            )
+            if not accepted:
                 if self.local_type == NodeType.FULL_NODE and not is_localhost(self.peer_info.host):
                     self.log.error(
                         f"Peer has been rate limited and will be disconnected: {self.peer_info.host}, "
-                        f"message: {message_type}"
+                        f"message: {message_type}, "
+                        f"{limiter_msg}"
                     )
```

Comment:
As above, though it won't let me provide a commitable suggestion for the whole thing since not all lines were changed.

```suggestion
                    details = ", ".join([
                        f"{self.peer_info.host}",
                        f"message: {message_type}"
                        f"{limiter_msg}",
                    ])
                    self.log.error("Peer has been rate limited and will be disconnected: {details}")
```